### DISCUSSION TITLE
Highlight selectable Pie chart data

### DISCUSSION THREAD

**Discussion by kimuradev** [plouc/nivo#1655] [Q&A] [answered]
Is it possible to highlight one specific data when you click on this? Today I can see the data chart increasing the size when you put your mouse over. But I would like to highlight when I click and put an opacity color in the others. 

<img width="586" alt="Screen Shot 2021-07-02 at 09 56 05" src="https://user-images.githubusercontent.com/4400304/124277733-c45d9780-db1b-11eb-8ad6-4215b3d9e672.png">

**Comment by wyze** [comment, marked as answer]
You need to implement and replace the `arcs` layer. Here is an example: https://codesandbox.io/s/vibrant-wescoff-txxtm?file=/src/App.tsx

**Reply by kimuradev** [reply]
That's exactly what I'm looking for. Thanks!

**Reply by kimuradev** [reply]
@wyze is not possible to keep tooltip working with this solution? Because even if I try to use a custom tooltip such as [storybook](https://nivo.rocks/storybook/?path=/story/pie--custom-tooltip) suggest, it is not working anymore.
